moto/autoscaling/models.py

```diff
@@ -262,10 +258,17 @@ def set_desired_capacity(self, new_capacity):
         if self.desired_capacity > curr_instance_count:
             # Need more instances
             count_needed = int(self.desired_capacity) - \
-                int(curr_instance_count)
+                           int(curr_instance_count)
```

Comment:
Yeah I have a fix queued up, as well as a fix for a test that I didn't realize was failing - should be pushing soon.